### DISCUSSION TITLE
posts/%EC%9A%B0%EB%B6%84%ED%88%AC-%EC%84%9C%EB%B2%84-root-%EB%B9%84%EB%B0%80%EB%B2%88%ED%98%B8-%EC%83%9D%EC%84%B1/

### DISCUSSION THREAD

**Discussion by giscus** [zhyunk/zhyunk#6] [General]
# posts/%EC%9A%B0%EB%B6%84%ED%88%AC-%EC%84%9C%EB%B2%84-root-%EB%B9%84%EB%B0%80%EB%B2%88%ED%98%B8-%EC%83%9D%EC%84%B1/

OS : Ubuntu 22.04.4 LTS

https://study.zhyun.kim/posts/%EC%9A%B0%EB%B6%84%ED%88%AC-%EC%84%9C%EB%B2%84-root-%EB%B9%84%EB%B0%80%EB%B2%88%ED%98%B8-%EC%83%9D%EC%84%B1/

**Comment by zhyunk** [comment]
Ubuntu Desktop 참고 
- <https://www.sangchul.kr/1180>
